README.md

```diff
@@ -1,15 +1,10 @@
 # stactools-spot
 
-Template repostitory for [stactools](https://github.com/stac-utils/stactools) packages.
+A subpackage of stactools for working with [SPOT](https://open.canada.ca/data/en/dataset/d799c202-603d-4e5c-b1eb-d058803f80f9) data.
 
-## How to use
+This subpackage converts the [Geobase Index shapefile](http://ftp.maps.canada.ca/pub/nrcan_rncan/image/spot/geobase_orthoimages/index/GeoBase_Orthoimage_Index.zip), that describes the SPOT image extents, to a STAC. 
```

Comment:
Does it output a STAC [item](https://github.com/radiantearth/stac-spec/blob/master/item-spec/item-spec.md), [catalog](https://github.com/radiantearth/stac-spec/blob/master/catalog-spec/catalog-spec.md), or [collection](https://github.com/radiantearth/stac-spec/blob/master/collection-spec/collection-spec.md)?